core/tests/protractor_utils/workflow.js

```diff
@@ -58,7 +60,21 @@ var createExplorationAndStartTutorial = function() {
   var creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage;
   creatorDashboardPage.get();
   // Wait for the dashboard to transition the creator into the editor page.
-  creatorDashboardPage.clickCreateActivityButton();
+  users.isAdmin().then(function(result) {
+    creatorDashboardPage.clickCreateActivityButton();
+    if (result) {
+      var testCreateModal = element(by.css('.protractor-test-creation-modal'));
```

Comment:
`testCreateModal` is a the real modal users will see, not a dummy, right? Prefixing the name with `test` could be confusing, especially since we usually don't include the `test` prefix (e.g. on line 73 with `createExplorationButton`).

Reply:
Updated